akka-http-core/src/main/scala/akka/http/scaladsl/util/FastFuture.scala

```diff
@@ -111,7 +111,7 @@ object FastFuture {
     else sequence(futures).fast.map(_.foldLeft(zero)(f))
 
   def reduce[T, R >: T](futures: TraversableOnce[Future[T]])(op: (R, T) ⇒ R)(implicit executor: ExecutionContext): Future[R] =
-    if (futures.isEmpty) failed(new NoSuchElementException("reduce attempted on empty collection"))
+    if (futures.isEmpty) failed(new scala.NoSuchElementException("reduce attempted on empty collection"))
```

Comment:
Please advice me of we have to suppress stack trace here also.

Reply:
Yes, sounds good.